suite-common/logger/src/logsSlice.ts

```diff
@@ -0,0 +1,42 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+export type LogEntry = { datetime: string; type: any; payload?: Record<any, any> };
+
+type LogsSliceState = {
+    logs: LogEntry[];
+};
```

Comment:
If there is no immediate plan to add something else to this slice, I would prefer to avoid `state.logs.logs`... Why not to keep it as it was – just an array? 
```suggestion
type LogsSliceState = LogEntry[];
```

If we plan to add something else, I would prefer to think about some different name i.e. state.logs.logEntries.

Reply:
If it's not a dealbreaker, I would rename it to `state.logs.logEntries`. I did this to follow the convention as have state slices object with it's properties. This way it's easily extendable if there is need to add anything and naming is not confusing. What do you think?

Reply:
https://github.com/trezor/trezor-suite/pull/7359/commits/3cc634793eec06a7461fbad15de709a91906b19b

Reply:
ok, not a big deal, state.logs.logEntries sound fine 👍